docs/server-backup-version-comparison.md

```diff
@@ -1,33 +1,30 @@
-<!--- will this be in 8.1?
- --->
-
 # Server version and backup version comparison
 
 A MySQL change to a feature, such as the structure of a redo log
 record, can cause older versions of Percona XtraBackup to fail. To
 ensure that you can back up and restore your data, use a Percona
-XtraBackup version that is equal to or above your source server version.
+XtraBackup version that is equal to your source server major version. This means if you use Percona XtraBackup 8.1.x, you can safely back up the source server from 8.1.x to 8.1.xx.
 
 !!! admonition "See also"
    
     [How XtraBackup works](how-xtrabackup-works.md)
 
 The `--no-server-version-check` option performs the following test.
-Before the backup starts, XtraBackup compares the source system version to
-the Percona XtraBackup version. If the source system version is greater
+Before the backup starts, XtraBackup compares the source server version to
+the Percona XtraBackup version. If the source server version is greater
```

Comment:
This is confusing.
In the first paragraph, we say they can safely back up the source server from 8.1.x t0 8.1.xx
Here, we say that if the source server is greater than the PXB version xtrabackup stops the backup process
At the end we say that if they override the check they can back up to 8.4.xx